### ISSUE TITLE
[BUG] Stuck "Connect Wallet" if view reload

### ISSUE THREAD

**Issue by NektoDron** [Chia-Network/chia-blockchain#2132]
**Describe the bug**
If plotting has been started ant wallet refresh stuck it on the 1.0.5 wallet. 

**To Reproduce**

1. Go to plotting tab
2. Start any plotting
3. Select View/Reload in the menu
4. See "Connecting wallet"
5. Open developer tools and see "Already connected, not reconnecting..."

**Expected behavior**
Should view reloaded.

**Desktop (please complete the following information):**

- OS: Windows 10
- CPU: Intel 3690k

**Additional context**

**Comment by peterzhen40** [comment]
the same
![image](https://user-images.githubusercontent.com/20519831/115264152-a01f2b00-a168-11eb-96e4-61357894b953.png)

**Comment by StrikeNP** [comment]
I also have this issue. 
Specs:
Chia version 1.0.5
Ryzen 9 5950x
64GB Ram
RTX 3090
m.2 nvme temp drive

My console outputs the same messages as Peter

**Comment by OddyAte** [comment]
I have the same issue. My plotting showed Plotting 100% with no activity in Log so I tried a reload, now it's stuck and I am not sure if the plotting will finish.

Chia version 1.0.5
Ryzen 7 3700X
16GB Ram
GTX 1070
Windows 10 Pro

**Comment by dexter74** [comment]
Fix: https://github.com/Chia-Network/chia-blockchain/issues/2148
Delete Folder : C:\Users\<username>\.chia

cmd:
```
set USER=MYUSERNAME
"C:\Users\%USER%\AppData\Local\chia-blockchain\app-1.0.5\resources\app.asar.unpacked\daemon\daemon.exe"
"C:\Users\%USER%\AppData\Local\chia-blockchain\Chia.exe"
```

**Comment by brunobliss** [comment]
> 
> 
> Fix: #2148
> Delete Folder : C:\Users<username>.chia
> 
> cmd:
> 
> ```
> set USER=MYUSERNAME
> "C:\Users\%USER%\AppData\Local\chia-blockchain\app-1.0.5\resources\app.asar.unpacked\daemon\daemon.exe"
> "C:\Users\%USER%\AppData\Local\chia-blockchain\Chia.exe"
> ```

If this requires a reboot of the app and daemon won't it also stop the plotting process? What about the blockchain download ?

**Comment by StrikeNP** [comment]
> If this requires a reboot of the app and daemon won't it also stop the plotting process? What about the blockchain download ?

Yes, it will stop the plotting process. Today I discovered that my gui had fixed itself after being broken, so there's hope if you just let it sit. I'd say it was broken for ~4hrs before it fixed itself this time. The only difference I'm aware of is when it broke this time it was on 1.1.0 instead of 1.0.5
